src/applications/gi-sandbox/components/content/AllCampusesModalContent.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export default function StudentComplaintsModalContent() {
```

Comment:
You have another StudentComplaintsModalContent. You may want to rename one of them.

Reply:
- Thanks!